### PR TITLE
Fix startvs.cmd for path with space

### LINKED ISSUE #15387: Problem executing startsvs.cmd
### Describe the bug
Im trying to run the script startsvs.cmd located on
C:\Users\Miguel Morales\AspNetCore\src\Mvc
But im getting error that says
"C\Users\Miguel not found"
It must be because the " " character.
I tryied adding "" or '', but not working.
Also tryied to edit the script but I am not being able to make it work

### To Reproduce
I've cloned the version from the git.
Im following some tutorial on plutalsight to learn more about asp.net core.
So when I try the command  startvs the error happends

### Screenshots
test11

### Additional context
script

The error
error

I still have lot to learn lol

### PR DESCRIPTION
**Pull request by Kahbazi**
Fixes #15387

### CODE REVIEW

File: startvs.cmd [hunk 1]
```diff
@@ -13,7 +13,7 @@ SET DOTNET_MULTILEVEL_LOOKUP=0
 :: Put our local dotnet.exe on PATH first so Visual Studio knows which one to use
 SET PATH=%DOTNET_ROOT%;%PATH%
 
-SET sln=%1
+SET sln=%~1
```
Comment (dougbu) [inline startvs.cmd]: Please explain

File: startvs.cmd [hunk 2]
```diff
@@ -27,4 +27,4 @@ IF NOT EXIST "%DOTNET_ROOT%\dotnet.exe" (
     exit /b 1
 )
 
-start %sln%
+start "" "%sln%"
```
Comment (dougbu) [inline startvs.cmd]: Quotes surrounding solution make sense but the extra argument to `start` does not.

**Comment by dougbu** [comment]
@aspnet/build please do real review and merge

File: startvs.cmd (hunk 1 above)
Comment (Kahbazi) [reply startvs.cmd]: https://ss64.com/nt/syntax-args.html

> %~1   Expand %1 removing any surrounding quotes (")

File: startvs.cmd (hunk 2 above)
Comment (Kahbazi) [reply startvs.cmd]: according to [docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/start#syntax) the first double quotes goes for the Title

**Comment by dougbu** [comment]
@Kahbazi have you tested this locally? Looks fine but I would appreciate your confirmation before we merge

**Review by dougbu — APPROVED** [review]

**Comment by Kahbazi** [comment]
@dougbu yes, it works on my machine.

**Comment by dougbu** [comment]
Thanks @Kahbazi ❕

**Pull request merged**